src/seeded_state.rs

```diff
@@ -1,11 +1,14 @@
+#[cfg(feature = "std")]
 use std::collections::{HashMap, HashSet};
```

Comment:
nitpick: since those are only used once, could you use paths instead of imports? (i.e. `type ... = std::...`)

Reply:
Ok, changed it